string directory_name
---
sensor_msgs/Image[] images